fellow-submissions/mark-griffith/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link rel="stylesheet" href="https://unpkg.com/blocks.css/dist/blocks.min.css" />
+    <title>The True Test</title>
+</head>
+<body>
+    <main>
+        <section>
+            <header>
+                <h1>The True Test For Greatness</h1>
+                <p>Answer this correctly for a prize!</p>
+            </header>
+            <fieldset>
+                <legend>Games</legend>
+                <label for="#">What game is the question below referencing:
+                    <input type="radio" name="game" value="Skyrim"> Skyrim
+                    <input type="radio" name="game" value="Fortnite"> Fortnite
+                    <input type="radio" name="game" value="Minecraft"> Minecraft
```

Comment:
Each `radio` input needs a label. Example:
```html
<input type="radio" name="game" value="Skyrim" id="skyrim">
<label for="skyrim">Skyrim</label>
```
And remember, the `for` attribute for your label should match the `id` on your `<input>`.